main.tf

```diff
@@ -162,3 +162,10 @@ resource "random_string" "session_secret" {
   length  = 32
   special = false
 }
+
+output "asg_outputs" {
+  value = {
+    asg_id   = "${aws_autoscaling_group.kong.id}"
+    asg_name = "${aws_autoscaling_group.kong.name}"
+  }
+}
```

Comment:
```suggestion
output "asg_outputs" {
  value = {
    asg_id   = aws_autoscaling_group.kong.id
    asg_name = aws_autoscaling_group.kong.name
  }
}

```

Reply:
updated the script to output the ASG objet

Reply:
Also moved ASG output block to output.tf